stackstorm-jinja/actions/render_file.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: "render_file"
+runner_type: "run-python"
+description: "Render template"
+enabled: true
+entry_point: "render_file.py"
+parameters:
+  template_pack:
+    type: "string"
+    description: "Path to the template"
+    required: true
+  template_path:
+    type: "string"
+    description: "Path to the template"
```

Comment:
Maybe we should assume a dir, like `templates`, and make `path` a relative path inside that dir? Since we're striving for consistency and all that :)